daemon/cmd/daemon_main.go

```diff
@@ -857,6 +857,10 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.Bool(option.DNSProxyEnableTransparentMode, defaults.DNSProxyEnableTransparentMode, "Enable DNS proxy transparent mode")
 	option.BindEnv(vp, option.DNSProxyEnableTransparentMode)
 
+	flags.Bool(option.DNSProxyInsecureSkipTransparentModeCheck, false, "Allows DNS proxy transparent mode to be disabled even if encryption is enabled. Enabling this flag will cause proxied DNS traffic to leave the node unencrypted.")
```

Comment:
```suggestion
	flags.Bool(option.DNSProxyInsecureSkipTransparentModeCheck, false, "Allows DNS proxy transparent mode to be disabled even if encryption is enabled. Enabling this flag can cause proxied DNS traffic to leave the node unencrypted.")
```

nit: enabling this flag doesn't _cause_ it, it merely allows it. Not too relevant, though.